policy_machine.gemspec

```diff
@@ -33,5 +33,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency('byebug')
   s.add_development_dependency('neography', '~> 1.1')
   s.add_development_dependency('database_cleaner')
-  s.add_development_dependency('rails', '~> 6.0')
+  s.add_development_dependency('rails', '~> 6.1')
```

Comment:
why the bump? asking cause im wondering if this is indeed a development dependency or actually a runtime dependency

Reply:
Because `ActiveRecord::Base.connection_db_config.config` is logging from Rails 6.1
> DEPRECATION WARNING: DatabaseConfig#config will be removed in 6.2.0 in favor of DatabaseConfigurations#configuration_hash which returns a hash with symbol keys (called from irb_binding at (irb):20)

Reply:
That or check for either method.